styleguide/components/hero/hero.css

```diff
@@ -47,7 +47,7 @@
   background-image: url(mds-components.jpeg);
 }
 
-.soul {
+.brand-identity {
```

Comment:
There are people missing in the splash image due to this change. This will need to be fixed before merging